### PR TITLE
Add locales validator

### PR DESCRIPTION
**Pull request by slavaleleka**
https://github.com/AdguardTeam/FiltersCompiler/issues/94

### CODE REVIEW

**Comment by slavaleleka** [comment]
adguard-filters-compiler should be updated to 1.1.3 here (when it's ready)

File: locales/en/filters.json [hunk 1]
```diff
@@ -66,5 +66,249 @@
 	{
 		"filter.224.name": "AdGuard Chinese filter",
 		"filter.224.description": "EasyList China + AdGuard Chinese filter. Filter list that specifically removes ads on websites in Chinese language."
+	},
+	{
+		"filter.101.name": "EasyList",
+		"filter.101.description": "EasyList is the primary subscription that removes adverts from web pages in English language. Already included in AdGuard Base filter."
```
Comment (el-termikael) [inline locales/en/filters.json:72]: in English language —> in the English language или  in English

**Review by el-termikael — APPROVED** [review]

**Review by ameshkov — APPROVED** [review]

File: filters/ThirdParty/filter_237_PolishAnnoyance/metadata.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 237,
-  "name": "Polskie Filtry Elementów Irytujących",
-  "description": "Filter list that hides and blocks pop-ups, widgets, newsletters, push notifications, arrows, tagged internal links that are off-topic, and other irritating elements. Polskie Filtry Ciasteczkowe is already in it.",
+  "name": "Polish Annoyance Filters",
```
Comment (el-termikael) [inline filters/ThirdParty/filter_237_PolishAnnoyance/metadata.json:3]: Annoyances 
(по аналогии с AdGuard Annoyances filter и Fanboy's Annoyances)

File: locales/en/filters.json [hunk 3]
```diff
@@ -66,5 +66,249 @@
...
+	{
+		"filter.218.name": "Estonian List",
+		"filter.218.description": "Filter for ad blocking on Estonian sites."
+	},
+	{
+		"filter.219.name": "ChinaList+EasyList",
+		"filter.219.description": "Adblocking list for China. Note that it does not actually include EasyList."
+	},
+	{
+		"filter.220.name": "CJX's Annoyance List",
```
Comment (el-termikael) [inline locales/en/filters.json:219]: Annoyances

File: locales/en/filters.json [hunk 4]
```diff
@@ -66,5 +66,249 @@
...
+	{
+		"filter.235.name": "Iranian filter",
+		"filter.235.description": "Ad blocking rules for Iranian/Persian websites."
+	},
+	{
+		"filter.236.name": "road-block",
+		"filter.236.description": "Romanian ad blocking filter subscription."
+	},
+	{
+		"filter.237.name": "Polish Annoyance Filters",
```
Comment (el-termikael) [inline locales/en/filters.json:267]: Annoyances

File: locales/en/filters.json [hunk 5]
```diff
@@ -66,5 +66,249 @@
...
+		"filter.238.name": "Polish Anti Adblock Filters",
+		"filter.238.description": "Official Polish filters against Adblock alerts."
+	},
+	{
+		"filter.239.name": "Fanboy's Anti-thirdparty Fonts",
+		"filter.239.description": "A filter that blocks third-party fonts. It may break the look and design of some websites."
+	},
+	{
+		"filter.240.name": "BarbBlock",
+		"filter.240.description": "BarbBlock is a content blocking list with standalone browser extensions for Chrome and Firefox. It blacklists sites which have used DMCA takedowns to force removal from other content blocking lists. Such takedowns are categorically invalid, but they can be effective at intimidating small open-source projects into compliance."
```
Comment (el-termikael) [inline locales/en/filters.json:280]: which have used --> that have used

**Review by el-termikael — CHANGES_REQUESTED** [review]

**Review by el-termikael — APPROVED** [review]

**Review by el-termikael — APPROVED** [review]

**Review by el-termikael — APPROVED** [review]

**Review by el-termikael — APPROVED** [review]

**Review by ameshkov — APPROVED** [review]

**Pull request merged**